### PR TITLE
remove extra space in svg class attribute

### LINKED ISSUE #1701: extra space in svg class attribute
### Package

- [X] lucide
- [ ] lucide-angular
- [ ] lucide-flutter
- [ ] lucide-preact
- [X] lucide-react
- [ ] lucide-react-native
- [ ] lucide-solid
- [ ] lucide-svelte
- [ ] lucide-vue
- [ ] lucide-vue-next
- [ ] Figma plugin
- [ ] source/main
- [ ] other/not relevant

### Version

0.293.0

### Browser

- [ ] Chrome/Chromium
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] iOS Safari
- [ ] Opera
- [ ] Other/not relevant

### Operating system

- [ ] Windows
- [ ] Linux
- [ ] macOS
- [ ] Other/not relevant

### Description

we have some snapshot tests and they started to show extra space at the end on this classname:
image

not a big problem, but if anyone could easily remove the space please do trim that

### Steps to reproduce

just render and see the html

### Checklist

- [X] I have searched if someone has submitted a similar issue before and there weren't any. (Please make sure to also search closed issues, as this issue might already have been resolved.)

### PR DESCRIPTION
**Pull request by Sukomal07**
fixes #1701

### CODE REVIEW

**Comment by ericfennis** [comment]
Conflicts

**Comment by Sukomal07** [comment]
> Conflicts

resolved

File: packages/lucide-react/src/createLucideIcon.ts [hunk 1]
```diff
@@ -39,16 +50,14 @@ const createLucideIcon = (iconName: string, iconNode: IconNode): LucideIcon => {
         },
         [
           ...iconNode.map(([tag, attrs]) => createElement(tag, attrs)),
-          ...(
-            (Array.isArray(children) ? children : [children]) || []
-          )
-        ],
-      ),
+          ...((Array.isArray(children) ? children : [children]) || []),
```
Comment (jguddas) [inline packages/lucide-react/src/createLucideIcon.ts:53]: `Array.isArray(children) ? children : [children]` is always `truthy`.

```suggestion
          ...(Array.isArray(children) ? children : [children]),
```

**Review by jguddas — CHANGES_REQUESTED** [review]

**Review by ericfennis — APPROVED** [review]
Awesome!

**Pull request merged**